README.md

```diff
@@ -1 +1,177 @@
-# book-store
\ No newline at end of file
+<a name="readme-top"></a>
+<div align="center">
+
+  <h1><b>Bookstore</b></h1>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Bookstore ](#-Bookstore-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+      - [🔭 Future Features ](#-future-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [🚀 Project Presentation Video ](#-project-presentation-video-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Testing](#testing)
+    - [Deployment](#deployment)
+  - [👥 Authors](#-authors)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Bookstore <a name="about-project"></a>
+
+>  The Bookstore is a website biulet with react and redux and create an MVP version of it that allows us to:
+
+Display a list of books,
+Add a book,
+Remove a selected book.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Languages</summary>
+  <ul>
+    <li>CSS</li>
+    <li>React</li>
+  </ul>
+</details>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Github</li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+- **Display a list of books**
+- **Add a book**
+- **Remove a selected booke**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+#### 🔭 Future Features <a name="future-features"></a>
+
+- **Add a book**
+- **Remove a selected booke**
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- <a href="#" target="_blank">Live Demo Link</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Project Presentation Video <a name="live-demo"></a>
+
+- <a href=""</a>
```

Comment:
- [ ] Kindly remove this section from your readme since you have no video presentation.